ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/presenter/PermanentIdRequestPresenter.java

```diff
@@ -0,0 +1,156 @@
+                IplantAnnouncer.getInstance()
+                               .schedule(new ErrorAnnouncementConfig(appearance.requestLoadFailure()));
+            }
+
+            @Override
+            public void onSuccess(String result) {
+                final AutoBean<PermanentIdRequestList> decode = AutoBeanCodex.decode(factory,
+                                                                                     PermanentIdRequestList.class,
+                                                                                     result);
+
+                view.loadRequests(decode.as().getRequests());
+            }
+        });
+
+    }
+
+    @Override
+    public void loadPermIdRequests() {
+        getPermIdRequests();
+    }
+
+    @Override
+    public void updateRequest(final PermanentIdRequestUpdate update) {
+        view.mask(I18N.DISPLAY.loadingMask());
+        if (selectedRequest != null && update != null) {
+            prsvc.updatePermanentIdRequestStatus(selectedRequest.getId(),
+                                                 update,
+                                                 new AsyncCallback<String>() {
+
+                                                     @Override
+                                                     public void onFailure(Throwable caught) {
+                                                         view.unmask();
+                                                         IplantAnnouncer.getInstance()
+                                                                        .schedule(new ErrorAnnouncementConfig(appearance.statusUpdateFailure()));
+
+                                                     }
+
+                                                     @Override
+                                                     public void onSuccess(String result) {
+                                                         view.unmask();
+                                                         IplantAnnouncer.getInstance()
+                                                                        .schedule(new SuccessAnnouncementConfig(appearance.statusUpdateSuccess()));
+                                                         selectedRequest.setStatus(update.getStatus());
+                                                         view.update(selectedRequest);
+                                                     }
+                                                 });
+        }
+    }
+
+    @Override
+    public void createPermanentId() {
+        view.mask(I18N.DISPLAY.loadingMask());
+        if (selectedRequest != null) {
+            prsvc.createPermanentId(selectedRequest.getId(), new AsyncCallback<String>() {
+
+                @Override
+                public void onFailure(Throwable caught) {
+                    view.unmask();
+                    IplantAnnouncer.getInstance()
+                                   .schedule(new ErrorAnnouncementConfig(appearance.createPermIdFailure()));
```

Comment:
This error should probably be displayed inside an error dialog, so the admin user can view the error message returned by our service endpoint, since this could be an error message from the EZID API that our endpoint returns to the UI in a JSON error message.